Mergin/plugin.py

```diff
@@ -551,6 +551,7 @@ def remove_local_project(self):
         settings = QSettings()
         settings.remove(f"Mergin/localProjects/{self.project_name}")
         self.parent().reload()
+        iface.newProjectCreated.emit()
```

Comment:
Is this a correct place, should not we emit after cur_proj.clear? Ideally with a comment why we need to do it.